Classes/Service/SearchEngineNotifier.php

```diff
@@ -21,7 +21,7 @@
 /*
  * Service to communicate with the search engine
  */
-class SearchEngineService implements LoggerAwareInterface
+class SearchEngineNotifier implements LoggerAwareInterface
```

Comment:
You have moved that file vom directory Service to Notifier. No problem, but in that case rename the directory to "Notifier", too.